### DISCUSSION TITLE
Support for Python 2.7

### DISCUSSION THREAD

**Discussion by okaljuvee** [jpype-project/jpype#1093] [Q&A] [answered]
What's the latest version of jpype that supports Python 2.7?  I've looked through https://jpype.readthedocs.io/en/latest/CHANGELOG.html but there is no explicit mention of dropping support for 2.7; and there is a mention of still supporting Python 2.7 in 0.7.0.

**Comment by Thrameos** [comment, marked as answer]
I believe we dropped it immediately after the Python 2.7 retirement date.   So 0.7.1 would have been the last one with Python 2 support.   After that the core was reworked to convert it to CPython which required Python 3.
